packages/ra-ui-materialui/src/button/DeleteWithConfirmButton.tsx

```diff
@@ -131,19 +133,3 @@ DeleteWithConfirmButton.propTypes = {
     icon: PropTypes.element,
     translateOptions: PropTypes.object,
 };
-
-const PREFIX = 'RaDeleteWithConfirmButton';
-
-const StyledButton = styled(Button, {
-    name: PREFIX,
-    overridesResolver: (props, styles) => styles.root,
-})(({ theme }) => ({
-    color: theme.palette.error.main,
-    '&:hover': {
-        backgroundColor: alpha(theme.palette.error.main, 0.12),
-        // Reset on mouse devices
-        '@media (hover: none)': {
-            backgroundColor: 'transparent',
-        },
-    },
-}));
```

Comment:
I couldn't find any case where the removal of this code would have any impact, but I'm still wondering why we had it before.
@djhi  Do you have any clues? Can we remove it safely?

Reply:
I think this is something we use to do for older versions of MUI